### ISSUE TITLE
Ubuntu 20.04 will not launch correctly from Start Menu

### ISSUE THREAD

**Issue by garyanson** [microsoft/WSL#7173]
### Windows Build Number

Microsoft Windows [Version 10.0.21390.2025]

### WSL Version

- [X] WSL 2
- [ ] WSL 1

### Kernel Version

5.10.43

### Distro Version

Ubuntu 20.04

### Repro Steps

Click on Start
Type in Ubuntu
Click on the Ubuntu 20.04 menu entry
Terminal Window Opens with Ubuntu 20.04 LTS title
The following is displayed:

![image_2021-07-13_092540](https://user-images.githubusercontent.com/27372147/125489680-1650a871-51ec-4317-aee0-1d257118987b.png)

The parameter is incorrect.
Press any key to continue...

I have done the changes to .wslconfig regarding the gui (enabled=true/false) and no changes to the outcome

### Expected Behavior

A bash prompt

### Actual Behavior

The parameter is incorrect.
Press any key to continue...

**Comment by craigloewen-msft** [comment]
Could you please file logs for this? This will help us determine what's going on and what could be causing this error.

/logs

**Comment by garyanson** [comment]
I've attached the log file as directed...
[wsl.zip](https://github.com/microsoft/WSL/files/6811966/wsl.zip)

**Comment by craigloewen-msft** [comment]
Is Windows Terminal set as your default Terminal Type? 

If you change it back to Windows Console host do you still see this error?

If yes then this bug is already tracked, and is fixed and will be updated in an upcoming version!

**Comment by garyanson** [comment]
Hi

**Comment by garyanson** [comment]
Hi, Yes Windows Terminal was the default Terminal Type however there was another entry on the settings that I believe actually was the culprit. The Default termi9nal application as apposed to the Default profile.
In my case the Default terminal application was "Windows Terminal Preview, Microsoft Corporation 1.9.1523.0" which I changed to Windows Console Host. After that it did not matter whcih Default profile I chose the "Ubuntu 20.04 LTS" start menu item worked as expected.
